src/database/models.py

```diff
@@ -0,0 +1,27 @@
+# Third Party
+from sqlalchemy import Column, ForeignKey, Integer, LargeBinary, String
+from sqlalchemy.ext.declarative import declarative_base
+
+# initialize automapping
+Base = declarative_base()
+
+
+class Document(Base):
+    __tablename__ = 'document'
+
+    id = Column(String, primary_key=True)
+    name = Column(String)
+    pages_number = Column(Integer)
```

Comment:
would it be possible to count this number dynamically?

Reply:
It would be, but I believe it would be also slower as we would need to access Page table and do the aggregation (and for that particular endpoint where we actually use this value it is not need to fetch the data from Page table). I can imagine that there could be SQLAlchemy function / expression that would automatically do it for us however if it would not update it when inserting the pages then it would be probably slower anyway. It also depends on the usecases, if we would like to somehow more manipulate with the pages then it would make sense to have it dynamically, however while this is just once generated and that it is basically read-only, then calculating the value once and have it kinda "cached" makes sense to me.